projects/demo/src/modules/experimental/card/examples/5/index.less

```diff
@@ -0,0 +1,83 @@
+:host {
+    display: flex;
+    gap: 1.2rem;
+    min-width: 40.625rem;
+}
+
+.badge {
+    background: #aff218;
+    color: #333;
+}
+
+.money {
+    background: #ffdd2d;
+    color: #333;
+}
+
+.poster[data-surface] {
```

Comment:
Let's try to get rid of these artificial specificity bumps here as well.

Reply:
fixed